sdks/python/apache_beam/runners/dataflow/internal/clients/dataflow/dataflow_v1b3_messages.py

```diff
@@ -2649,16 +2649,19 @@ class MapTask(_messages.Message):
   the list before any instructions which depends on its output.
 
   Fields:
+    counterPrefix: Counter prefix that can be used to prefix counters. Not
+      currently used in Dataflow.
     instructions: The instructions in the MapTask.
     stageName: System-defined name of the stage containing this MapTask.
       Unique across the workflow.
     systemName: System-defined name of this MapTask. Unique across the
       workflow.
   """
 
-  instructions = _messages.MessageField('ParallelInstruction', 1, repeated=True)
```

Comment:
Did the field numbering change?

Reply:
I do not think this matter. E.g. see [`LeaseWorkItemRequest`](https://github.com/apache/beam/pull/6662/files/32c080d61e767c0a84fdd5507c3275348bdf7132#diff-d0fe331100a13c8f759cfe53569e2a78R2582) also has the fields in a different order. If a Dataflow job passes, it should validate that the proto is fine, right?